tests/test/export/main.fmf

```diff
@@ -0,0 +1,4 @@
+summary: Test the feature to export tests to yaml or dict
+contact: Vector Li <huanli@redhat.com>
+test: ./test.sh
+framework: beakerlib
```

Comment:
The last two lines are not necessary, they are inherited from here:

https://github.com/teemtee/tmt/blob/4272d242da23f3d5dd5be8db1f683720c22ac72f/tests/main.fmf#L1

Reply:
Removed.

Reply:
Thanks.